src/main/kotlin/features/mining/PickaxeAbility.kt

```diff
@@ -170,6 +174,11 @@ object PickaxeAbility : FirmamentFeature {
 		nowAvailable.useMatch(it.unformattedString) {
 			val ability = group("name")
 			lastUsage[ability] = TimeMark.farPast()
+			if (!TConfig.cooldownReadyToast) return
+			val mc: MinecraftClient = MinecraftClient.getInstance()
+			mc.toastManager.add(
+				SystemToast.create(mc, SystemToast.Type.NARRATOR_TOGGLE, Text.of("Pickaxe Cooldown"), Text.of("Pickaxe ability is ready!"))
```

Comment:
Can you make use of the translate function to create translations instead of hardcoding text? I.e.:
```suggestion
				SystemToast.create(mc, SystemToast.Type.NARRATOR_TOGGLE, tr("firmament.pickaxe.ability-ready","Pickaxe Cooldown"), tr("firmament.pickaxe.ability-ready.desc", "Pickaxe ability is ready!"))
```